extensions/fairifier-rdf-extension/dtls/fairifier/DetectFileFormatCommand.java

```diff
@@ -48,24 +51,27 @@
  */
 public class DetectFileFormatCommand extends Command {
 
-    private static final RDFParser rdfParserttl = new TurtleParser();
-    private static final RDFParser rdfParserntriples = new NTriplesParser();
-    private static final RDFParser rdfParserrdfxml = new RDFXMLParser();
-    private static final RDFParser rdfParserJsonld = new JSONLDParser();
-    private static final RDFParser rdfParserNQuads = new NQuadsParser();
-    private static final RDFParser rdfParserJson = new RDFJSONParser();
-    private static final RDFParser rdfParserTrix = new TriXParser();
-    private static final RDFParser rdfParserTrig = new TriGParser();
-
-    private final static RDFParser[] parsers = { rdfParserntriples, rdfParserttl, rdfParserrdfxml, rdfParserJsonld,
-            rdfParserNQuads, rdfParserJson, rdfParserTrix, rdfParserTrig };
+    private final static ArrayList<RDFParser> parsers = new ArrayList<RDFParser>() {
```

Comment:
This is an unconventional way of initializing the arraylist. Better to create a static initialization block and add it in there.